### ISSUE TITLE
[BUG]: ERROR ~ Error executing process > 'NF_FLU:NANOPORE:MQC_VERSIONS_TABLE (1)'

### ISSUE THREAD

**Issue by Codes1985** [CFIA-NCFAD/nf-flu#87]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Bug/Issue

Using a locally-installed instance of [nf-flu v3.5.1 release f18f8ce](https://github.com/CFIA-NCFAD/nf-flu/releases/tag/3.5.1) to process Nanopore sequence data, and experienced an error during the multiqc file generation as described below.  Interestingly, I did not experience this during my recent testing, though I was laser-focussed on the Illumina branch.

### Nextflow command-line

```shell
nextflow /[REDACTED]/nf-flu_v3.5.1/cfia-ncfad-nf-flu-3.5.1/workflow/main.nf --input IRVC20241017IHN_surv.csv --platform nanopore --clair3_user_variant_model /[REDACTED]/clair3_models/rerio/clair3_models/r1041_e82_400bps_hac_v420/ --outdir IRVC20241017IHN_surv_nf-flu_results --major_allele_fraction 0.50 --minor_allele_fraction 0.25 -profile singularity,slurm -resume
```

### Error Message

```shell
[04/c429ca] NF_…ORE:MQC_VERSIONS_TABLE (1) | 2 of 2, failed: 2, retries: 1 ✘
Plus 1 more processes waiting for tasks…
Oops... Pipeline execution stopped with the following message: [23:04:53] - INFO: mqc_versions_table 0.2.0
[23:04:53] - INFO: Output directory '.' already exists.
stream.nim(105)          next
Error: unhandled exception: Invalid indentation: got 1, expected 0 [YamlStreamError]
[35/728ac9] NOTE: Process `NF_FLU:NANOPORE:MQC_VERSIONS_TABLE (1)` terminated with an error exit status (1) -- Execution is retried (1)
ERROR ~ Error executing process > 'NF_FLU:NANOPORE:MQC_VERSIONS_TABLE (1)'

Caused by:
  Process `NF_FLU:NANOPORE:MQC_VERSIONS_TABLE (1)` terminated with an error exit status (1)


Command executed:

  mqc_versions_table \
    --versions-yaml collated_versions.yml \
    --nextflow-version 24.04.4 \
    --workflow-name CFIA-NCFAD/nf-flu \
    --workflow-version 3.5.1

Command exit status:
  1

Command output:
  (empty)

Command error:
  [23:04:53] - INFO: mqc_versions_table 0.2.0
  [23:04:53] - INFO: Output directory '.' already exists.
  stream.nim(105)          next
  Error: unhandled exception: Invalid indentation: got 1, expected 0 [YamlStreamError]

Work dir:
  /[REDACTED]/work/04/c429ca9ceda7b1e8ac685aec177ec4

Tip: you can try to figure out what's wrong by changing to the process work dir and showing the script file named `.command.sh`

 -- Check '.nextflow.log' file for details
```

### Workflow Version

nf-flu v3.5.1; revision f18f8ce

### Nextflow Executor

slurm

### Nextflow Version

24.04.4

### Hardware

HPC Cluster

### Operating System (OS)

Distributor ID: CentOS Description:    CentOS Linux release 7.9.2009 (Core) Release: 7.9.2009 Codename:  Core

### Conda/Container Engine

Singularity

**Comment by peterk87** [comment]
Thanks for creating the issue @Codes1985 . Can you attach the `yml` files from the `work/04/c429ca9ceda7b1e8ac685aec177ec4` directory? It sounds like it's an indentation issue with one of the processes... 

I'm tempted to move towards CSV output for the version info since that's what anyone would be looking at in the end.

**Comment by Codes1985** [comment]
[collated_versions.txt](https://github.com/user-attachments/files/17438361/collated_versions.txt)
Apologies for the delayed respone; please see attached.

Please note, the `yml` file from the `work/04/c429ca9ceda7b1e8ac685aec177ec4` directory was a symlink to the attached file. I had to change `yml` to `txt` for uploading purposes.

Thanks you!

**Comment by peterk87** [comment]
Something funny is happening with the CLAIR3 process. 

```
 "NF_FLU:NANOPORE:CLAIR3":
     clair3: 1.0.10
     samtools: 1.15.1 
 END_VERSIONS
```
should be
```
"NF_FLU:NANOPORE:CLAIR3":
    clair3: 1.0.10
    samtools: 1.15.1 
```

Would you happen to be able to share a `.command.sh` from a CLAIR3 process execution in your work dir? The only thing I can see right now is the use of a custom Rerio model for Clair3, but I'm not sure how that would be affecting the indentation.

**Comment by Codes1985** [comment]
[command.txt](https://github.com/user-attachments/files/17438840/command.txt)

That does seem odd.

**Comment by peterk87** [comment]
This should be fixed in 3.5.2. If not, let know me!

**Comment by Codes1985** [comment]
Hi @peterk87!  Apologies for not replying to this thread; yes, this is resolved and working well!
